### PR TITLE
Introduce "orchard pause" and "orchard resume" commands

### PR DESCRIPTION
**Pull request by edigaryev**
Resolves https://github.com/cirruslabs/orchard/issues/45.

### CODE REVIEW

File: internal/command/pause/worker.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import (
+	"context"
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var wait uint64
+
+func newPauseVMCommand() *cobra.Command {
```
Comment (fkorotkov) [inline internal/command/pause/worker.go:12]:
```suggestion
func newPauseWorkerCommand() *cobra.Command {
```

File: internal/command/pause/worker.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		Args:  cobra.ExactArgs(1),
+	}
+
+	command.PersistentFlags().Uint64Var(&wait, "wait", 0,
+		"wait the specified amount of seconds for the worker to stop running any VMs")
+
+	return command
+}
+
+func runPauseVM(cmd *cobra.Command, args []string) error {
```
Comment (fkorotkov) [inline internal/command/pause/worker.go:26]:
```suggestion
func runPauseWorker(cmd *cobra.Command, args []string) error {
```

File: internal/command/pause/worker.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		subCtx, cancel := context.WithTimeout(cmd.Context(), time.Duration(wait)*time.Second)
+		defer cancel()
+
+		for {
+			vms, err := client.VMs().FindForWorker(cmd.Context(), worker.Name)
+			if err != nil {
+				return err
+			}
+
+			if len(vms) == 0 {
```
Comment (fkorotkov) [inline internal/command/pause/worker.go:58]: Let's check only for VMs not in final state. If all the VMs are stopped it's OK to continue.

File: internal/command/resume/worker.go [hunk 4]
```diff
@@ -0,0 +1,42 @@
+package resume
+
+import (
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func newResumeVMCommand() *cobra.Command {
```
Comment (fkorotkov) [inline internal/command/resume/worker.go:8]:
```suggestion
func newResumeWorkerCommand() *cobra.Command {
```

File: internal/command/resume/worker.go [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+		Use:   "worker NAME",
+		Short: "Resume a worker",
+		RunE:  runResumeVM,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	return command
+}
+
+func runResumeVM(cmd *cobra.Command, args []string) error {
```
Comment (fkorotkov) [inline internal/command/resume/worker.go:19]:
```suggestion
func runResumeWorker(cmd *cobra.Command, args []string) error {
```

File: pkg/resource/v1/worker.go [hunk 6]
```diff
@@ -9,6 +9,8 @@ type Worker struct {
 
 	MachineID string
 
+	Unschedulable bool
```
Comment (fkorotkov) [inline pkg/resource/v1/worker.go:12]: What do yo think of `SchedulingPaused` name instead?

File: internal/command/pause/worker.go (hunk 1 above)
Comment (edigaryev) [reply internal/command/pause/worker.go]: Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.

File: internal/command/pause/worker.go (hunk 2 above)
Comment (edigaryev) [reply internal/command/pause/worker.go]: Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.

File: internal/command/resume/worker.go (hunk 4 above)
Comment (edigaryev) [reply internal/command/resume/worker.go]: Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.

File: internal/command/resume/worker.go (hunk 5 above)
Comment (edigaryev) [reply internal/command/resume/worker.go]: Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.

File: pkg/resource/v1/worker.go (hunk 6 above)
Comment (edigaryev) [reply pkg/resource/v1/worker.go]: I'm fine with it, renamed in https://github.com/cirruslabs/orchard/pull/73/commits/0a29be3254d4d91cb18fdc5b9aca0eef48928517.

File: internal/command/pause/worker.go (hunk 3 above)
Comment (edigaryev) [reply internal/command/pause/worker.go]: Good catch, fixed in https://github.com/cirruslabs/orchard/pull/73/commits/ccbdc767d2b4177667dd8adc7610392794c83565.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**